02_03_hw2 NEW/ProximaNova/stylesheet.css

```diff
@@ -0,0 +1,463 @@
+	src: url('ProximaNovaCond-Extrabld.eot');
+	src: local('Proxima Nova Condensed Extrabold'), local('ProximaNovaCond-Extrabld'),
+		url('ProximaNovaCond-Extrabld.eot?#iefix') format('embedded-opentype'),
+		url('ProximaNovaCond-Extrabld.woff') format('woff'),
+		url('ProximaNovaCond-Extrabld.ttf') format('truetype');
+	font-weight: 800;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Proxima Nova ExCn Th';
+	src: url('ProximaNovaExCn-ExtrabldIt.eot');
+	src: local('Proxima Nova Extra Condensed Extrabold Italic'), local('ProximaNovaExCn-ExtrabldIt'),
+		url('ProximaNovaExCn-ExtrabldIt.eot?#iefix') format('embedded-opentype'),
+		url('ProximaNovaExCn-ExtrabldIt.woff') format('woff'),
+		url('ProximaNovaExCn-ExtrabldIt.ttf') format('truetype');
+	font-weight: 800;
+	font-style: italic;
+}
+
+@font-face {
+	font-family: 'Proxima Nova ExCn Bl';
+	src: url('ProximaNovaExCn-BlackIt.eot');
+	src: local('Proxima Nova Extra Condensed Black Italic'), local('ProximaNovaExCn-BlackIt'),
+		url('ProximaNovaExCn-BlackIt.eot?#iefix') format('embedded-opentype'),
+		url('ProximaNovaExCn-BlackIt.woff') format('woff'),
+		url('ProximaNovaExCn-BlackIt.ttf') format('truetype');
+	font-weight: 900;
+	font-style: italic;
+}
+
+@font-face {
+	font-family: 'Proxima Nova ExCn Lt';
+	src: url('ProximaNovaExCn-LightIt.eot');
+	src: local('Proxima Nova Extra Condensed Light Italic'), local('ProximaNovaExCn-LightIt'),
+		url('ProximaNovaExCn-LightIt.eot?#iefix') format('embedded-opentype'),
+		url('ProximaNovaExCn-LightIt.woff') format('woff'),
+		url('ProximaNovaExCn-LightIt.ttf') format('truetype');
+	font-weight: 300;
+	font-style: italic;
+}
+
+@font-face {
+	font-family: 'Proxima Nova Cn Th';
+	src: url('ProximaNovaCond-ThinIt.eot');
+	src: local('Proxima Nova Condensed Thin Italic'), local('ProximaNovaCond-ThinIt'),
+		url('ProximaNovaCond-ThinIt.eot?#iefix') format('embedded-opentype'),
+		url('ProximaNovaCond-ThinIt.woff') format('woff'),
+		url('ProximaNovaCond-ThinIt.ttf') format('truetype');
+	font-weight: 100;
+	font-style: italic;
+}
+
+@font-face {
+	font-family: 'Proxima Nova ExCn Lt';
+	src: url('ProximaNovaExCn-Light.eot');
+	src: local('Proxima Nova Extra Condensed Light'), local('ProximaNovaExCn-Light'),
+		url('ProximaNovaExCn-Light.eot?#iefix') format('embedded-opentype'),
+		url('ProximaNovaExCn-Light.woff') format('woff'),
+		url('ProximaNovaExCn-Light.ttf') format('truetype');
+	font-weight: 300;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Proxima Nova Cn Rg';
+	src: url('ProximaNovaCond-BoldIt.eot');
+	src: local('Proxima Nova Condensed Bold Italic'), local('ProximaNovaCond-BoldIt'),
+		url('ProximaNovaCond-BoldIt.eot?#iefix') format('embedded-opentype'),
+		url('ProximaNovaCond-BoldIt.woff') format('woff'),
+		url('ProximaNovaCond-BoldIt.ttf') format('truetype');
+	font-weight: bold;
+	font-style: italic;
+}
+
+@font-face {
+	font-family: 'Proxima Nova ExCn Th';
+	src: url('ProximaNovaExCn-Extrabld.eot');
```

Comment:
This won't work since line 281 will rewrite. And this is doubled (`src`). And this gets repeated a lot in code. Please, check